webpack.config.ts

```diff
@@ -161,13 +165,28 @@ export const clientConfig: Configuration = {
     }
 };
 
+const assetsTemplateCss = new CleanCSS().minify(renederedItemsAssetsCss).styles.trim()
+const assetHash = (asset: string): string =>
+    createHash('sha256').update(asset).digest('base64');
+
```

Comment:
Unfortunately couldn't get this to work from `server/csp.ts` and spent enough time trying to make it happen. Kind of means there was not much of a point to do this PR with TypeScript but at least we have a TS webpack config now.